mono/mini/genmdesc.c

```diff
@@ -220,7 +220,7 @@ init_table (void) {
 
 static void
 output_char (FILE *f, char c) {
-	if (isalnum (c))
+	if (isalnum ((unsigned char)c)) // Cast to unsigned char to work around a bug in MSVCRT in debug builds
```

Comment:
I don't think this is a bug in MSVCRT, from [MSDN](https://msdn.microsoft.com/en-us/library/k84c0490.aspx):
> The behavior of isalnum and _isalnum_l is undefined if c is not EOF or in the range 0 through 0xFF, inclusive. When a debug CRT library is used and c is not one of these values, the functions raise an assertion.

So it seems we're getting a value < -1 here?